lib/service/lokal-hamburk.js

```diff
@@ -23,19 +23,22 @@ module.exports = function () {
   var processMenu = function (obj, options, next) {
     var items = []
 
-    // only interested in Polévky (1), Speciality Lokálu (2), Hlavní jídla (3)
-    for (var j = 1; j < 4; j++) {
+    // only interested in Polévky (1), Lehká a bezmasá jídla (2), Speciality Lokálu (3), Hlavní jídla (4)
+    for (var j = 1; j < 5; j++) {
       var lunchMenu = obj.menus[j].menu
       for (var i = 0; i < lunchMenu.length; i++) {
         if (!lunchMenu[i].item) { // not interested in headers
           continue
         }
-
-        items.push({
+        var item = {
           item: getItem(lunchMenu[i]),
           price: lunchMenu[i].price,
           amount: getAmount(lunchMenu[i])
-        })
+        }
+        if(item.item.match(/^Expres/))
+          items.unshift(item) //Expres first
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '{' and instead saw 'items'.](https://www.codacy.com/app/Socialbakers/luncher/file/2374639910/issues/source?bid=3218200&fileBranchId=3224190#l39)